tests/test_issues/input/linkml_issue_2224.yaml

```diff
@@ -0,0 +1,62 @@
+id: DJControllerSchema
+name: DJControllerSchema
+title: LinkML schema for my DJ controller
+imports:
+- linkml:types
+classes:
+  DJController:
+    slots:
+    - jog_wheels
+    - tempo
+    - volume_faders
+    - crossfaders
+    slot_usage:
+      tempo:
+        examples:
+        - value: 120.0
+        - value: 144.0
+        - value: 126.8
+        - value: 102.6
+        annotations:
+          expected_value: a number between 0 and 300
+          preferred_unit: BPM
```

Comment:
only override one of `examples` or `annotations` here, test that the non-overridden one has its values propagated from the original slot definition

Reply:
Makes sense! i'll remove the overrides for `annotations` (retaining the overrides for `examples` only).

Will check that the original `annotations` from `tempo` are being propagated as expected:

```yaml
    annotations:
      expected_value: a number between 0 and 200
      preferred_unit: BPM
```